README.md

```diff
@@ -1,2 +1,70 @@
-# scraped_page
+# ScrapedPage
+
 Write declarative scrapers in Ruby
+
+## Usage
+
+Create a subclass of `ScrapedPage` for each page you wish to scrape.
```

Comment:
This is a bit ambiguous. If I'm going to scrape 500 MP pages, I presumably don't need 500 subclasses…